CoreFoundation/NumberDate.subproj/CFTimeZone.c

```diff
@@ -679,14 +679,10 @@ CFTimeZoneRef CFTimeZoneCreateWithWindowsName(CFAllocatorRef allocator, CFString
     return retval;
 }
 
-extern CFStringRef _CFGetWindowsAppleSystemLibraryDirectory(void);
 static void __InitTZStrings(void) {
     static CFLock_t __CFTZDirLock = CFLockInit;
     __CFLock(&__CFTZDirLock);
-    if (!__tzZoneInfo) {
-        CFStringRef winDir = _CFGetWindowsAppleSystemLibraryDirectory();
-        __tzZoneInfo = CFStringCreateWithFormat(NULL, NULL, CFSTR("%@\\etc\\zoneinfo"), winDir);
-    }   
+    // TODO(compnerd) figure out how to initialize __tzZoneInfo
```

Comment:
I think we'll need to make sure this works before merging right?

Reply:
So, the interesting thing is that there is `__CFCopyWindowsTimeZoneList` which will enumerate the data from the registry.  We reference that in `CFTimeZoneCopyKnownNames`.  I don't have the context on why CoreFoundation attempted to provide its own copy of the timezone data rather than using the system one on Windows (as it already uses the system provided one on Darwin, BSD, and Linux).

Reply:
I think our Windows version of CF used to bring its own, but using the system TZ data is almost certainly better, since it can change underneath us.